content/en/device-provisioning-service/client-library.md

```diff
@@ -50,49 +51,75 @@ void plgd_dps_shutdown(void);
 plgd_dps_context_t *plgd_dps_get_context(size_t device);
 
 /// @brief Set DPS manager callbacks.
-void plgd_dps_manager_init_callbacks(plgd_dps_context_t *ctx, plgd_dps_on_status_change_cb_t on_change_cb,
-                                     void *on_change_data, oc_cloud_cb_t on_cloud_change_cb,
-                                     void *on_cloud_change_data);
+void plgd_dps_set_manager_callbacks(plgd_dps_context_t *ctx, plgd_dps_on_status_change_cb_t on_change_cb,
+  void *on_change_data, oc_cloud_cb_t on_cloud_change_cb, void *on_cloud_change_data);
```

Comment:
Please describe these callbacks. I don't fully understand what and when occurs on_change_data , on_cloud_change and so on. Only onStatusChanged is clear because of the example below. It should be described directly in the header file.